test/IDE/print_set_accessor.swift

```diff
@@ -0,0 +1,6 @@
+public struct SomeType {
```

Comment:
This is sort of relevant to this change because I added `UserInaccessible` to `__setter_access`, but it is actually accessible. Turned out not to matter anyway since access control is printed differently, thought it'd be nice to add a test for it though.